pallets/ah-migrator/src/lib.rs

```diff
@@ -256,6 +273,23 @@ pub mod pallet {
 		type SendXcm: SendXcm;
 		/// Weight information for extrinsics in this pallet.
 		type AhWeightInfo: WeightInfo;
+		/// Asset Hub Treasury accounts migrating to the new treasury account address (same account
+		/// address that was used on the Relay Chain).
+		///
+		/// The asset list should not include the native asset.
```

Comment:
And these are all assets that should be migrated?

Reply:
I will add the doc the clarify what AssetId is

Reply:
it actually depends from `Assets` impl, it can be u32 or xcm location, both would work

Reply:
updated docs anyway